### PR TITLE
fix: fixing weird vertical shift

### LINKED ISSUE #3592: Event Dashboard: Weird vertical displacement on Unpublishing Event
**Is your feature request related to a problem? Please describe.**

 Weird vertical displacement on `Unpublishing Event`

**Describe the solution you'd like**

Fix the displacement of the page .

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3592 

#### Short description of what this resolves:
ezgif com-gif-maker (4)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
@kushthedude  last time i didn't used length this time i am using "gt" helper is {{unverifed-user-message.length}} is greater than zero then the div will show up .

**Comment by maze-runnar** [comment]
please check it . As if i am also wrong this time too even i am thinking it is working :smile:

File: app/templates/events/view.hbs [hunk 1]
```diff
@@ -1,5 +1,4 @@
-{{#if (or (eq session.currentRouteName 'events.view.index') (eq session.currentRouteName 'events.view.export'))}}
-  {{#if (eq model.state 'draft')}}
```
Comment (kushthedude) [inline app/templates/events/view.hbs]: Revert this

Comment (maze-runnar) [reply app/templates/events/view.hbs]: i don't get it .

> Revert this

I don't get it . please explain a bit.

Comment (kushthedude) [reply app/templates/events/view.hbs]: `{{#if (eq model.state 'draft')}}`
Let it be on next line only

**Comment by kushthedude** [comment]
Upload a GIF showing the transition with latest code commit.

**Comment by maze-runnar** [comment]
ezgif com-gif-maker

**Comment by maze-runnar** [comment]
@kushthedude @uds5501 please review .

**Pull request merged**